internal/control_plane/command/root.go

```diff
@@ -0,0 +1,63 @@
+package command
+
+import (
+	"octavius/internal/control_plane/command/start"
+	"octavius/internal/logger"
+	"os"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+var rootCmd = &cobra.Command{
+	Use:   "ControlPlane",
+	Short: "Control plane of octavius",
+	Long:  `Control plane of octavius takes request from cli`,
+}
+
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		logger.Error("error in execute", err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+	logger.Setup()
+	logger.Info("in controlPlane root init")
+	cobra.OnInitialize(initConfig)
+
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/config.json)")
+
+	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+
+	// add your client commands here
+	rootCmd.AddCommand(start.GetCmd())
+}
+
+func initConfig() {
+	logger.Info("in controlPlane root init config")
+	// Use config file from the flag.
+	if cfgFile != "" {
+		viper.SetConfigFile(cfgFile)
+	} else {
+		home, err := homedir.Dir()
+		if err != nil {
+			logger.Error("CP Init config issue : ", err)
+			os.Exit(1)
+		}
+		logger.Info("home: " + home)
```

Comment:
Please remove this log as this has no extra information, this looks like for debugging 😝 .